src/App/component/AddProjectDialog.jsx

```diff
@@ -15,32 +13,22 @@ import {
 
 export default function AddProjectDialog({open, reloadProjects, handleClose}) {
   const [projectName, setProjectName] = useState("")
-  const [githubRepositoryURL, setGithubRepositoryURL] = useState("")
-  const [sonarRepositoryURL, setSonarRepositoryURL] = useState("")
-  const setIsGithubAvailable = useState(false)[1]
-  const setIsSonarAvailable = useState(false)[1]
   const jwtToken = localStorage.getItem("jwtToken")
 
   const createProject = () => {
     let checker = []
-    if (projectName === "" || (githubRepositoryURL === "" && sonarRepositoryURL === "")) {
+    if (projectName === "") {
       alert("不準啦馬的>///<")
     } else {
```

Comment:
Suggest use Early-Return to solve the unexpected `projectName` input cases, and also add a `trim()` method to the `projectName` in the if statement for filter some white-space-like input cases.

```suggestion
    if (projectName.trim() === "") {
      alert("不準啦馬的>///<")
      return
    } 
```